dist-workspace.toml

```diff
@@ -0,0 +1,22 @@
+[workspace]
+members = ["cargo:."]
+
+# Config for 'dist'
+[dist]
+# The preferred dist version to use in CI (Cargo.toml SemVer syntax)
+cargo-dist-version = "0.27.0"
+# CI backends to support
+ci = "github"
+# The installers to generate for each app
+installers = ["shell", "powershell", "msi"]
+# Target platforms to build apps for (Rust target-triple syntax)
+targets = ["aarch64-apple-darwin", "aarch64-unknown-linux-gnu", "x86_64-apple-darwin", "x86_64-unknown-linux-musl", "x86_64-pc-windows-msvc"]
+# Whether dist should create a Github Release or use an existing draft
+create-release = false
+# Path that installers should place binaries in
+install-path = "~/.svix/bin"
+# Whether to install an updater program
+install-updater = true
+# Whether CI should trigger releases with dispatches instead of tag pushes
+dispatch-releases = true
```

Comment:
If we later decide we want to automatically build the cli on tag push, this is what we'd tweak. I elected to use a manual trigger to start so we have a bit more control over it.